### PR TITLE
Added Mobile stepper to live code example

### PR DESCRIPTION
**Pull request by shubham-eaton**
#### Changes proposed in this Pull Request:

- Changes done for converting Mobile stepper to live code example

#### Screenshots / Screen Recording (if applicable)

-

#### To Test:

- yarn docs-start

### CODE REVIEW

File: docs/src/componentDocs/MobileStepper/markdown/MobileStepperExamples.mdx [hunk 1]
```diff
@@ -1,86 +1,18 @@
...
-import MobileStepperDotVariant from '../images/mobileStepper_dot.png';
-import MobileStepperTextVariant from '../images/mobileStepper_text.png';
-import MobileStepperProgressVariant from '../images/mobileStepper_progress.png';
+import {MobileStepperDotVariant} from '../examples/MobileStepperDotVariant'
+import {MobileStepperTextVariant} from '../examples/MobileStepperTextVariant'
+import {MobileStepperProgressVariant} from '../examples/MobileStepperProgressVariant'
 
 ## Mobile Stepper
 
-The basic mobile stepper uses dots to indicate progress.
```
Comment (surajeaton) [inline docs/src/componentDocs/MobileStepper/markdown/MobileStepperExamples.mdx:8]: You need to keep this description for mobile stepper.

File: docs/src/componentDocs/MobileStepper/markdown/MobileStepperExamples.mdx [hunk 2]
```diff
@@ -1,86 +1,18 @@
...
-<MobileStepper
-  activeStep={activeStep}
-  steps={steps}
-  leftButton={<BackButton />}
-  rightButton={<NextButton />}
-  variant={'dots'}
-/>;
-```
 
 ## Using Text / Progress Bar
```
Comment (surajeaton) [inline docs/src/componentDocs/MobileStepper/markdown/MobileStepperExamples.mdx:10]: Follow the convention:

```
## Heading

Description Text
<Example />
```

So, there will be 2 separate examples:

```
## Using Text
Description of text
<ExampleText />

## Using Progress Bar
Description of progress bar
<ExampleProgress />

File: docs/src/componentDocs/MobileStepper/examples/MobileStepperProgressVariantExample.tsx [hunk 3]
```diff
@@ -0,0 +1,36 @@
...
+            style={{width: 100, alignSelf: 'flex-end'}}
+            onPress={() => ({})}
+            mode="contained">
+            Next
+          </Button>
+        </View>
+      }
+      variant={'progress'}
+    />
+    ;
```
Comment (surajeaton) [inline docs/src/componentDocs/MobileStepper/examples/MobileStepperProgressVariantExample.tsx:34]: Same comment as above

File: docs/src/componentDocs/MobileStepper/examples/MobileStepperTextVariantExample.tsx [hunk 4]
```diff
@@ -0,0 +1,31 @@
...
+        <Button
+          onPress={() => ({})}
+          mode="contained"
+          style={{width: 100, alignSelf: 'flex-end', marginLeft: 40}}>
+          Next
+        </Button>
+      }
+      variant={'text'}
+    />
+    ;
```
Comment (surajeaton) [inline docs/src/componentDocs/MobileStepper/examples/MobileStepperTextVariantExample.tsx:29]: Same comment as above

File: docs/src/componentDocs/MobileStepper/markdown/MobileStepperExamples.mdx (hunk 1 above)
Comment (shubham-eaton) [reply docs/src/componentDocs/MobileStepper/markdown/MobileStepperExamples.mdx:8]: Added the description

File: docs/src/componentDocs/MobileStepper/markdown/MobileStepperExamples.mdx (hunk 2 above)
Comment (shubham-eaton) [reply docs/src/componentDocs/MobileStepper/markdown/MobileStepperExamples.mdx]: Changes done as per suggestion

File: docs/src/componentDocs/MobileStepper/examples/MobileStepperDotVariant.tsx [hunk 5]
```diff
@@ -0,0 +1,37 @@
...
+          Next
+        </Button>
+      }
+      variant={'dots'}
+    />`;
+
+export const MobileStepperDotVariant = (): JSX.Element => (
+  <Box>
+    <MobileStepperDotVariantExample />
+    <CodeBlock code={codeSnippet} language="jsx" />
```
Comment (JeffGreiner-eaton) [inline docs/src/componentDocs/MobileStepper/examples/MobileStepperDotVariant.tsx:31]: add dataLine="20"

File: docs/src/componentDocs/MobileStepper/examples/MobileStepperProgressVariant.tsx [hunk 6]
```diff
@@ -0,0 +1,41 @@
...
+        </Button>
+        </View>
+      }
+      variant={'progress'}
+    />`;
+
+export const MobileStepperProgressVariant = (): JSX.Element => (
+  <Box>
+    <MobileStepperProgressVariantExample />
+    <CodeBlock code={codeSnippet} language="jsx" />
```
Comment (JeffGreiner-eaton) [inline docs/src/componentDocs/MobileStepper/examples/MobileStepperProgressVariant.tsx:35]: add dataLine="24"

File: docs/src/componentDocs/MobileStepper/examples/MobileStepperTextVariant.tsx [hunk 7]
```diff
@@ -0,0 +1,37 @@
...
+          Next
+        </Button>
+      }
+      variant={'text'}
+    />`;
+
+export const MobileStepperTextVariant = (): JSX.Element => (
+  <Box>
+    <MobileStepperTextVariantExample />
+    <CodeBlock code={codeSnippet} language="jsx" />
```
Comment (JeffGreiner-eaton) [inline docs/src/componentDocs/MobileStepper/examples/MobileStepperTextVariant.tsx:31]: add dataLine="20"

**Comment by JeffGreiner-eaton** [comment]
The text variant is having issue with flex 0 1 0 and needs to use flexBasis: 'auto'

image

We might need to have this updated in the react-native component library and publish an updated beta version as we did for a couple of other issues like this that we noticed https://github.com/etn-ccis/blui-react-native-component-library/pull/580/files

File: docs/src/componentDocs/MobileStepper/examples/MobileStepperDotVariant.tsx (hunk 5 above)
Comment (shubham-eaton) [reply docs/src/componentDocs/MobileStepper/examples/MobileStepperDotVariant.tsx]: Added as per suggestion

File: docs/src/componentDocs/MobileStepper/examples/MobileStepperProgressVariant.tsx (hunk 6 above)
Comment (shubham-eaton) [reply docs/src/componentDocs/MobileStepper/examples/MobileStepperProgressVariant.tsx]: Added as per suggestion

File: docs/src/componentDocs/MobileStepper/examples/MobileStepperTextVariant.tsx (hunk 7 above)
Comment (shubham-eaton) [reply docs/src/componentDocs/MobileStepper/examples/MobileStepperTextVariant.tsx]: Added as per suggestion

**Comment by shubham-eaton** [comment]
> The text variant is having issue with flex 0 1 0 and needs to use flexBasis: 'auto'
> 
> image
> 
> We might need to have this updated in the react-native component library and publish an updated beta version as we did for a couple of other issues like this that we noticed https://github.com/etn-ccis/blui-react-native-component-library/pull/580/files

Raised an issue to fix this in react-native-component-library
https://github.com/etn-ccis/blui-react-native-component-library/issues/583

**Review by JeffGreiner-eaton — APPROVED** [review]

**Pull request merged**